pkg/trace/config/apply.go

```diff
@@ -358,6 +368,62 @@ func (c *AgentConfig) applyDatadogConfig() error {
 		MaxRequestBytes: c.MaxRequestBytes,
 	}
 
+	// Instrumentation Telemetry config
+	c.TelemetryConfig = &TelemetryConfig{
+		Enabled: config.Datadog.GetBool("apm_config.telemetry.enabled"),
+	}
+	if c.TelemetryConfig.Enabled {
+		site := config.Datadog.GetString("site")
```

Comment:
I'm 100% sure there has to already be a helper method for this. If not, let's make one. https://github.com/DataDog/datadog-agent/blob/2373feabd39cf4148fa0fcc7fb2235bebb2dc220/pkg/trace/config/apply.go#L219-L227

Reply:
There was config.GetMainEndpoint function I was able to use